kratos/tests/materials_with_subproperties.json

```diff
@@ -0,0 +1,105 @@
+{
+    "properties" : [{
+        "model_part_name" : "Inlets",
+        "properties_id"   : 1,
+        "Material"        : {
+            "constitutive_law" : {
+                "name" : "HyperElastic3DLaw",
+                "name_after_merge_rule_of_mixtures" : "RuleOfMixturesLaw",
+                "combination_factors"      : [0.4, 0.3, 0.3 ]
+            },
+            "Variables"        : {
+            },
+            "Tables"           : {}
+        },
+        "sub_properties" : [{
+            "properties_id"   : 11,
+            "Material"        : {
+                "constitutive_law" : {
+                    "name" : "HyperElastic3DLaw"
+                },
+                "Variables"        : {
+                    "THICKNESS"     : 0.000889,
+                    "DENSITY"       : 7850.0,
+                    "YOUNG_MODULUS" : 206900000000.0,
+                    "POISSON_RATIO" : 0.29
+                },
+                "Tables"           : {}
+            },"sub_properties" : [{
+                "properties_id"   : 21,
+                "Material"        : {
+                    "constitutive_law" : {
+                        "name" : "HyperElastic3DLaw"
+                    },
+                    "Variables"        : {
+                        "THICKNESS"     : 0.000889,
+                        "DENSITY"       : 7850.0,
+                        "YOUNG_MODULUS" : 206900000000.0,
+                        "POISSON_RATIO" : 0.29
+                    },
+                    "Tables"           : {}
+                }
+            },{
+                "properties_id"   : 22,
+                "Material"        : {
+                    "constitutive_law" : {
+                        "name" : "LinearElastic3DLaw"
+                    },
+                    "Variables"        : {
+                        "THICKNESS"     : 0.000889,
+                        "DENSITY"       : 7850.0,
+                        "YOUNG_MODULUS" : 206900000000000.0,
+                        "POISSON_RATIO" : 0.0
+                    },
+                    "Tables"           : {}
+                }
+            },{
+                "properties_id"   : 2
```

Comment:
It would be better to use `"existing_properties_id"`

Reply:
Another option is to hava a flag says `"use_existing"` which is false by default

Reply:
I think the current interface is more general. You don't need to write anything special to consider already existing properties